third_party/llvm-project/dwarf2yaml.cpp

```diff
@@ -425,10 +428,12 @@ std::error_code dwarf2yaml(DWARFContext &DCtx, DWARFYAML::Data &Y) {
   dumpDebugAbbrev(DCtx, Y);
   dumpDebugStrings(DCtx, Y);
   dumpDebugARanges(DCtx, Y);
-  dumpDebugRanges(DCtx, Y); // XXX BINARYEN
-  dumpDebugLoc(DCtx, Y); // XXX BINARYEN
   dumpDebugPubSections(DCtx, Y);
   dumpDebugInfo(DCtx, Y);
   dumpDebugLines(DCtx, Y);
+  // These go last, because dumpDebugLoc relies on the address size being known
+  // from dumpDebugInfo.
```

Comment:
the comment explains Loc, but why is Ranges also moved?

Reply:
I thought it made sense to keep em together, but I can move just loc..